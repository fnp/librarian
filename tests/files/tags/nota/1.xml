<autor_utworu>Henryk Sienkiewicz</autor_utworu>
<nazwa_utworu>Latarnik</nazwa_utworu>

<nota><akap>Opowiadanie to osnute jest na wypadku rzeczywistym, o którym w swoim czasie pisał J. Horain
   w jednej ze swoich korespondencyj z Ameryki.</akap></nota>
