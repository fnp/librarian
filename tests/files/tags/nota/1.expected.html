<div id="book-text">
<h1>
<span class="wl author" id="i1">Henryk Sienkiewicz</span><span class="wl title" id="i2">Latarnik</span>
</h1>
<div class="note"><p class="wl paragraph" id="i3">Opowiadanie to osnute jest na wypadku rzeczywistym, o którym w swoim czasie pisał J. Horain
   w jednej ze swoich korespondencyj z Ameryki.</p></div>
</div>
